meta/template.md

```diff
@@ -1,9 +1,8 @@
 # Title
```

Comment:
```suggestion
# ***Untitled***
```

Reply:
Sure, I don't mind it. 

We can even go aggressive and say 
***Title goes here***